llvm/lib/ExecutionEngine/IntelJITEvents/jitprofiling.c

```diff
@@ -19,7 +19,9 @@
 
 #if ITT_PLATFORM==ITT_PLATFORM_WIN
 #include <windows.h>
+#if !INTEL_CUSTOMIZATION
 #pragma optimize("", off)
+#endif // !INTEL_CUSTOMIZATION
```

Comment:
I'm not sure what this does, but I suggest committing this change to llorg instead.

Reply:
Done.